zerver/templatetags/app_filters.py

```diff
@@ -141,6 +140,11 @@ def render_markdown_path(
                 api_url=context["api_url"],
             ),
         ]
+    extensions = [
+        *extensions,
+        zerver.lib.markdown.tabbed_sections.makeExtension(),
+    ]
```

Comment:
Why does this need to move?

Reply:
This had to be moved later since the JS examples' extension first add the markdown function for tabs, which is then supposed to be parsed by the extension for tabs. Thus, it needed to be at last.